gogoproto/helper.go

```diff
@@ -242,7 +250,7 @@ func EnabledGoStringer(file *google_protobuf.FileDescriptorProto, message *googl
 }
 
 func HasGoGetters(file *google_protobuf.FileDescriptorProto, message *google_protobuf.DescriptorProto) bool {
-	return proto.GetBoolExtension(message.Options, E_GoprotoGetters, proto.GetBoolExtension(file.Options, E_GoprotoGettersAll, true))
+	return proto.GetBoolExtension(message.Options, E_GoprotoGetters, proto.GetBoolExtension(file.Options, E_GoprotoGettersAll, true)) && !IsDropTypeDeclaration(message)
```

Comment:
Why?
I mean if someone doesn't want gogetters they can turn them off, but otherwise there is a good chance they will still work?